components/eam/src/physics/crm/physpkg.F90

```diff
@@ -1,20 +1,26 @@
 module physpkg
   !-----------------------------------------------------------------------------
   ! Purpose: Provides the interface to MMF physics package
+  ! 
+  ! Method: 
+  !   Each parameterization should be implemented with this sequence of calls:
+  !    1)  Call physics interface
+  !    2)  Check energy
```

Comment:
I believe it is physics_update() and then check_energy_chng() .

Reply:
oh this comment is originally in master

Reply:
Good catch, I had just copied this, but I'll rewrite it.

Reply:
updated.